packages/editor/src/components/rich-text/tinymce.js

```diff
@@ -158,6 +159,7 @@ export default class TinyMCE extends Component {
 			browser_spellcheck: true,
 			entity_encoding: 'raw',
 			convert_urls: false,
+			verify_html: false,
```

Comment:
What is this exactly?

Reply:
Disables TinyMCE's parsing to verify HTML. I don't mind leaving it, but it would make initialisation a bit faster. Since we're setting raw HTML already, we don't need this to be verified.